README.md

```diff
@@ -28,11 +36,13 @@ Checkout the project and type `./gradlew build`. You find the executable JAR in
      java version "1.8.0_144"
      Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
      Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed model
-     $ git clone https://github.com/AlloyTools/org.alloytools.alloy.git
-     $ cd org.alloytools.alloy
+     $ git clone --recursive https://github.com/haslab/Electrum2.git
```

Comment:
The git url doesn't look right.  It should still be the old (org.alloytools.alloy) git url